terraform/project.tf

```
@@ -0,0 +1,31 @@
+variable "project_name" {}
```

Repeat for all variables:
```suggestion
variable "project_name" {
  type = string
}
```